apps/condo/domains/ticket/components/TicketForm/UpdateTicketForm.tsx

```diff
@@ -11,28 +11,39 @@ import { Ticket, TicketFile } from '@condo/domains/ticket/utils/clientSchema'
 import ActionBar from '@condo/domains/common/components/ActionBar'
 import { Loader } from '@condo/domains/common/components/Loader'
 import { REQUIRED_TICKET_FIELDS, TICKET_SOURCE_TYPES } from '@condo/domains/ticket/constants/common'
-import { FormResetButton } from '@condo/domains/common/components/FormResetButton'
 
 import { BaseTicketForm } from '../BaseTicketForm'
 import { ErrorsContainer } from '../BaseTicketForm/ErrorsContainer'
 
 export const ApplyChangesActionBar = ({ handleSave, isLoading }) => {
     const intl = useIntl()
     const ApplyChangesMessage = intl.formatMessage({ id: 'ApplyChanges' })
+    const CancelLabel = intl.formatMessage({ id: 'Cancel' })
+
+    const { push, query: { id } } = useRouter()
+    const onCancel = useCallback((resetFields) => {
+        resetFields()
+        push(`/ticket/${id}`)
+    }, [id, push])
 
     return (
         <Form.Item noStyle shouldUpdate>
             {
-                ({ getFieldsValue }) => {
+                ({ getFieldsValue, resetFields, isFieldsTouched }) => {
                     const { property, details, placeClassifier, categoryClassifier, deadline } = getFieldsValue(REQUIRED_TICKET_FIELDS)
                     const disabledCondition = !property || !details || !placeClassifier || !categoryClassifier || !deadline
-
+                    const isNotTouched = !isFieldsTouched()
                     return (
                         <ActionBar isFormActionBar>
-                            <FormResetButton
+                            <Button
+                                key='cancel'
+                                onClick={() => onCancel(resetFields)}
```

Comment:
Why not to store `resetFields` with closure pattern? It would be look like this
```suggestion
                                onClick={onCancel(resetFields)}
```

Reply:
<img width="923" alt="Screenshot 2022-08-24 at 13 44 07" src="https://user-images.githubusercontent.com/64303474/186373153-229dcb28-c3bc-4b89-a17b-cf425ba209c8.png">